src/tests/unit/components/search-autocomplete/SearchAutocomplete.spec.jsx

```
@@ -0,0 +1,85 @@
+const options = ['Apple', 'Banana', 'Cherry']
+
+describe('SearchAutocomplete', () => {
+  const mockSetSearch = vi.fn()
+  const mockOnSearchChange = vi.fn()
+
+  beforeEach(() => {
+    render(
+      <SearchAutocomplete
+        search=''
+        setSearch={mockSetSearch}
+        onSearchChange={mockOnSearchChange}
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)